samples/subsys/ipc/openamp_rsc_table/remote/stm32mp157c_dk2.overlay

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (c) 2019, STMICROLECTRONICS
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/ {
+	chosen {
+		/*
+		 * shared memory reserved for the inter-processor communication
+		 */
+		zephyr,ipc_shm = &mcusram3;
+	};
+
+	mcusram3: memory1@10040000 {
+		device_type = "memory";
```

Comment:
Please remove this line (see https://github.com/zephyrproject-rtos/zephyr/pull/17860).